Stratis.Bitcoin/Features/BlockStore/BlockStoreLoop.cs

```diff
@@ -14,6 +14,11 @@
 
 namespace Stratis.Bitcoin.Features.BlockStore
 {
+    /// <summary>
+    /// The BlockStoreLoop simultaneously finds and downloads blocks
+    /// and stores them in the BlockRepository
+    /// <see cref="DownloadAndStoreBlocks"/>
+    /// </summary>
     public sealed class BlockStoreLoop
     {
         internal readonly ConcurrentChain Chain;
```

Comment:
It really depends on who is reading the code. The obvious comment is good for the reader who is unfamiliar with the code. Why? To actually confirm to this reader that there is really nothing more to see. 

So here we have "Chain". When I first touched Stratis, I thought - this is chain, this must represent a block chain - i.e. the chain of blocks. But nope, this is chain of headers only. So Chain is not obvious. 

But other things can be obvious, but just the assurance to the reader that they are really obvious is valuable for the reader. I.e. the reader does not need to look for any deeper meaning or something.

So it may seem to you as overkill, but it should not harm - feel free to write one liners.